http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValuesSpec.scala

```diff
@@ -451,8 +451,8 @@ class BuildHelmChartValuesSpec extends AnyFlatSpecLike with LeonardoTestSuite {
       """replicaCount=1,""" +
       """nodeSelector.cloud\.google\.com/compute-class=Balanced,""" +
       """tolerations.enabled=true,""" +
-      """tolerations.keyValue=auser2,""" +
-      """nodeSelector.group=auser2,""" +
+      """tolerations.keyValue=leo_2b3b2b9ce842ab8b6a6c614cb1f9604bb8a0d502d1af49c526b72b10,""" +
+      """nodeSelector.group=leo_2b3b2b9ce842ab8b6a6c614cb1f9604bb8a0d502d1af49c526b72b10,""" +
```

Comment:
I don't know the context; where do these values come from?

Reply:
it's [this](https://github.com/DataBiosphere/leonardo/blob/f9df012ff6afdcbe79a2b3f85870feff6be9d86b/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala#L397-L402)

See [this](https://github.com/DataBiosphere/leonardo/blob/f9df012ff6afdcbe79a2b3f85870feff6be9d86b/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala#L397-L402) for the convo

Reply:
got it got it! if this is a predictable value (the hash of `userEmail2`) could this use a helper variable so it can have some context in a comment?

Reply:
I mean a comment in the unit test to make it clear why we expect this specific long hash.

I figured having this line be like `nodeselector.group={hashedEmail}` would be more readable, with `hashedEmail` in the setup of the test.